src/ScriptExecutor.php

```diff
@@ -43,7 +43,8 @@ public function __construct(Composer $composer, IOInterface $io, Options $option
      */
     public function execute(string $type, string $cmd): void
     {
-        if (null === $expandedCmd = $this->expandCmd($type, $this->options->expandTargetDir($cmd))) {
+        $translatedCmd = $this->options->expandTargetDir($cmd);
```

Comment:
@fabpot The method name is actually called "expandTargetDir", the method "expandCmd" is a different method which is actually being stored in $expandedCmd.